pegasus/sites.v3/code.org/views/course_curriculum.haml

```diff
@@ -18,8 +18,8 @@
 - elsif series == "2.0"
   %p
     = I18n.t("dashboard_#{course}")
-    %a{href: "https://curriculum.code.org/csf/#{course}", target: '_blank'}
+    %a{href: "https://studio.code.org/s/#{course}-2021", target: '_blank'}
```

Comment:
if we want to make this future-proof, this could become `https://studio.code.org/s/#{course}`, and as an added bonus you could merge it today and it will get updated automatically on hard launch day. the problem is that the same strategy doesn't work for standards yet. I would say, feel free to keep this PR open as is, and if we happen to implement redirecting for standards then we can update this to lose the `-2021` prefix throughout the PR.

Reply:
https://codedotorg.atlassian.net/browse/PLAT-1125

Reply:
I can update but we had talked about not wanting the redirect for standards because it might be part of the problem of people getting redirected and not knowing and ending up making progress in the wrong year. So it feels like we are going to have to come here to update the standards URL anyway so it might be fine to leave it with the year?

Reply:
ahh, I just realized this won't work for this year, but could be done immediately after hard launch to version-proof for next year.

Reply:
> I can update but we had talked about not wanting the redirect for standards because it might be part of the problem of people getting redirected and not knowing and ending up making progress in the wrong year. So it feels like we are going to have to come here to update the standards URL anyway so it might be fine to leave it with the year?

what we do not want is urls with version-specific years getting redirected to other version years. redirecting the non-versioned URL to a versioned one should be ok.

regardless, we should hold off on this until after hard launch this year, then future proof for next yera.

Reply:
The non-versioned URLs were the concern but I guess not on pages like this. It was a concern that if we put them in resources that hang around year to year like the curriculum guide then you could click a link in the 2019 curriculum guide and all of a sudden be in the 2021 curriculum